test/assert.js

```diff
@@ -237,3 +283,122 @@ QUnit.test( "raises, alias for throws", function( assert ) {
 		throw "my error";
 	});
 });
+
+QUnit.module( "failing assertions", {
+	beforeEach: function( assert ) {
+		assert.push = function( result, actual, expected, message ) {
+
+			// inverts the result so we can test failing assertions
+			QUnit.assert.push.call( assert, !result, actual, expected, message );
```

Comment:
I'd like to get rid of all references to `QUnit.assert` outside of QUnit's source code, except those adding custom assertions.

In this case it should also remove the need to use `call()`.